src/Language/PureScript/Make/Actions.hs

```diff
@@ -343,7 +343,7 @@ buildMakeActions outputDir filePathMap foreigns usePrefix =
   requiresForeign = not . null . CF.moduleForeign
 
   progress :: ProgressMessage -> Make ()
-  progress = liftIO . TIO.hPutStr stderr . (<> "\n") . renderProgressMessage "Compiling "
+  progress = liftIO . TIO.hPutStr stderr . (<> "\n") . renderProgressMessage "CompilingX5 "
```

Comment:
🤔

Reply:
Five times faster?

Reply:
It's how I keep track of which compiler is running where. `purs --version` just reports e.g. 0.15.4 and changing that breaks all of the tooling.